### ISSUE TITLE
ome_diagnostics and support_assist_collection under log_type

### ISSUE THREAD

**Issue by MallocArray** [dell/dellemc-openmanage-ansible-modules#309]
##### Summary

In all of Dell's applications and marketing, SupportAssist is one word.  In the module ome_diagnostics, under log_type, one of the option is "support_assist_collection" which treats it as two words with _ separating each one.  Ideally, the option should be named supportassist_collection, but since the module has been released with it being treated as two words, can it be made into an alias one way or the other, so supportassist_collection becomes a valid option?  Documentation would also need to be updated to use this as the name, even if support_assist_collection continues to be available.

##### Component Name

ome_diagnostics

##### Ansible Version

```paste below

```

##### iDRAC or OpenManage Enterprise version
OME-M 1.30.00

##### Playbook used

```paste below
https://github.com/dell/dellemc-openmanage-ansible-modules/blob/collections/docs/modules/ome_diagnostics.rst
```

##### Expected Results

log_type value named supportassist_collection as SupportAssist is only one word

##### Actual Results

value is named support_assist_collection, treating it as separate words

#### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) 
  to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, 
  they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by jagadeeshnv** [comment]
support_assist would be the snake_case of SupportAssist. Moreover this is the format recommended by Ansible though not mandatory.

**Comment by MallocArray** [comment]
Maybe it depends on what your definition of snake_case is, but according to Wikipedia:
https://en.wikipedia.org/wiki/Snake_case
each space is replaced with _

In this case, SupportAssist is always documented across Dell as a single word, so in my eyes, would not fall under the convention of putting an underscore between them, hence my consideration of supportassist_collection.

Ultimately, this is your decision, but wanted to bring this to your attention for another review.  If it is decided to keep it formatted as 3 separate words, this issue can be closed.

**Comment by anupamaloke** [comment]
@jagadeeshnv, I am also somewhat aligned with @MallocArray in that we should be using "supportassist" as a single word. It would be good to have another choice named ```supportassist_collection``` for the ```log_type``` module argument and may be alias it.

**Comment by MallocArray** [comment]
Addressed in 5.3.0
